mqtt/mqtt3/README.md

```diff
@@ -20,6 +20,31 @@ The crate is not published to crates.io yet. Please generate docs locally with `
 
 See the `examples/` directory for examples of a publisher and subscriber, and for how to set a will.
 
+# Fuzz testing
+
+The crate is coming with fuzz tests. 
+
+## Prerequisites
+
+```bash
+sudo apt install tmux binutils
+cargo install afl
+```
+
+## Run fuzzer
+
+```bash
+echo 'core' | sudo tee /proc/sys/kernel/core_pattern
```

Comment:
Add a comment why this is needed. Something like:

```sh
# afl requires dumps to be taken as quickly as possible, so configure kernel to just write the coredump instead of anything fancy.
```